javascript/javascript1/week1/index.js

```diff
@@ -12,7 +13,7 @@ dogYearFuture = 2027;
 dogYear = dogYearFuture - dogYearOfBirth;
 let shouldShowResultInDogYears = dogYear;
```

Comment:
Why you need to assisgn dogYear to shouldShowResultInDogYears?

Reply:
const dogYear = dogYearFuture - dogYearOfBirth;
let shouldShowResultInDogYears = dogYear;